Tasks/Buhayev/Task3/task3 .py

```diff
@@ -0,0 +1,86 @@
+import time
+def check_input(func):
+    def wrapper():
+        while True:
+            try:
+                string_width = func()
+                time.sleep(.2)
+                try:
+                    assert string_width > 34
+                except:
+                    
+                    print('\nSorry string width is to small\n')
+                    time.sleep(.2)
+                    continue
+
+                try:
+                    assert string_width < 51
+                except:
+                    print('\nSorry string width is to big\n')
+                    time.sleep(.2)
+                    continue
+                return string_width
+            except:
+                print('\nSorry invalid input, width must be digit\n')
+                time.sleep(.2)
+                continue
+    return wrapper
+
+@check_input
+def input_string_width():
+    return int(input('Please enter a string width (a digit between 35 and 55)\n:   '))
```

Comment:
it looks like a decoration for the sake of decoration, but the realization itself is great